specification/components/r4/schemas/endpoints/a042-request-pre-signed-url-for-file-download.yaml

```diff
@@ -0,0 +1,97 @@
+  This endpoint makes use of a HTTP 307 temporary redirect. It redirects the caller to a temporary location from which the file contents can be downloaded directly.
+
+  The temporary location is only valid for a short period of time and should be used immediately. 
+
+  Callers of this endpoint must ensure they:
+    - follow this redirect to retrieve the file
+    - do not cache the temporary location
+    - generate a new redirect each time the file is downloaded
+
+  See the Response HTTP 307 section for further information.
+
+  ## Important notes
+  A referral pathway in e-RS can be made up of more than one UBRN. For example: a referral is booked and seen in a general knee clinical assessment service (UBRN #1), and the service decides to onward refer to a more specialist knee meniscus service (UBRN #2). This would result in two UBRNs for the referral pathway. There may be additional related UBRNs if there are multiple onward referrals.
+
+  As such, it is important that all clinical information is obtained from across all the related UBRNs referenced in [[HYPERLINK_A005]]. You can do this using the following endpoints:
+    - [[HYPERLINK_A006]]
+    - [[HYPERLINK_A007]]
+
+  Note: It is possible that the initial UBRN may be the only one in the referral pathway to have clinical information and/or attachments associated.
+
+  ## Use case 
+  As an authenticated user
+
+  I need to retrieve a clinical attachment associated with a referral or advice request
+
+  So that I can assess its content and decide what further action may be needed.
+
+  ## Related endpoints
+    - [[HYPERLINK_A005]] to retrieve details of a referral. This includes references to clinical attachments, related referrals and other important data.
+    - [[HYPERLINK_A024]] to retrieve the summary of an advice and guidance request. This endpoint provides important contextual information about the advice and guidance request (e.g. the service/specialty to which advice has been requested, etc).
+    - [[HYPERLINK_A025]] to retrieve the advice and guidance conversation between the referring organisation and service providing organisation.
+    - [[HYPERLINK_A007]] to generate a PDF file that summarises clinical information for a referral.
+
+  ## Sandbox test scenarios
+  The sandbox for this endpoint is a simple implementation that only supports success cases.
+
+  Inline with the behaviour described in "Support for a temporary redirect", the sandbox will return a HTTP 307 temporary redirect. For simplicity, the temporary location will be static and never expire, unlike the live environment.
+
+  A successful response will always be returned, provided the Binary ID is in a valid format. The Availability Status of a file is not considered in the sandbox and a file will always be considered to be available for retrieval.
+
+  Successful responses will always return the same example PDF file.
+
+summary: Request pre-signed URL to download file from document store (A042, FHIR R4)
```

Comment:
Seems like the title was updated to be "Retrieve Attachment (A042, FHIR R4)"

Reply:
Done